client/src/app/utils/waves-selector.ts

```diff
@@ -56,14 +49,16 @@ export const getWavesWithStatus = (
     wave: MigrationWave,
     tickets: Ticket[]
   ): TicketStatus[] =>
-    wave.applications.map((application): TicketStatus => {
-      const matchingTicket = getTicketByApplication(tickets, application.id);
-      if (matchingTicket?.error) {
-        return "Error";
-      } else if (matchingTicket?.status) {
-        return matchingTicket.status;
-      } else return "";
-    });
+    wave.applications
+      .map((application): TicketStatus => {
+        const matchingTicket = getTicketByApplication(tickets, application.id);
+        if (matchingTicket?.error) {
+          return "Error";
+        } else if (matchingTicket?.status) {
+          return matchingTicket.status;
+        } else return "";
+      })
+      .filter((ticketStatus) => ticketStatus !== "");
```

Comment:
`.filiter(Boolean)` will remove anything that is falsey ("", 0, false, undefined, null)